core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1522,7 +1740,7 @@ protected function eliminateNestedPages($arrPages, $strTable=null, $blnSorting=f
 	 * @param string     $strLightboxId An optional lightbox ID
 	 * @param FilesModel $objModel      An optional files model
 	 */
-	public static function addImageToTemplate($objTemplate, $arrItem, $intMaxWidth=null, $strLightboxId=null, FilesModel $objModel=null)
+	public static function addImageToTemplate__old($objTemplate, $arrItem, $intMaxWidth=null, $strLightboxId=null, FilesModel $objModel=null)
```

Comment:
This seems to be a left-over?

Reply:
This is the old `addImageToTemplate` implementation - we should remove it as a last step before merging. The integration tests are written in a way that you can compare the new with the old implementation and verify the behaviour.

see https://github.com/contao/contao/pull/1753#pullrequestreview-447061811

Reply:
Finally dropped in 1b670e533d5c03d21673c0d7c32c0154714a8a15